image-create

```diff
@@ -121,8 +121,8 @@ class MachineBuilder:
         # avoid too long boot times -- they cause painfully long tests, and are a bug
         try:
             timeout = 30
-            if self.machine.image == "fedora-rawhide":
-                # except with fedora-rawhide, which is slow by design:
+            if self.machine.image in ["fedora-rawhide", "fedora-37"]:
+                # except with fedora-rawhide and fedora-37, which is slow by design:
```

Comment:
I'm objecting because this heavily penalizes our test runtime -- we have several tests which wait for rebooting, and allowing that to take 100s instead of 2s will quickly add up. I'd rather revert this hack completely and specially treat the problem (like a hanging unit -- `packagekit.service` is our biggest offender here!)

Reply:
Moreover, this is the wrong place. The refresh failure is on "waiting for machine poweroff timed out", which is the self.machine.stop() call on line 115 -- and that already has a timeout of 500s (> 8 minutes!).

Really, let's rather drop the fedora-rawhide exception here, and get that fixed properly.